globals.js

```diff
@@ -36,7 +36,7 @@ const CONTINUE_BUTTON_TEXT = "continue";
 // Duration of the fixation cross.
 const FIX_DUR = -1;
 // The buttons to terminate presentation of the fix cross
-const FIX_CHOICES = [32]; // 32 == " " a spacebar to continue
+const FIX_CHOICES = [' ']; // 32 == " " a spacebar to continue
```

Comment:
Perhaps remove the comment here as is now outdated and I think `'  '` speaks more for itself than `32` did.